recipes/gpi_core/build.sh

```diff
@@ -0,0 +1,27 @@
+#!/bin/bash
+
+set -x
+
+UNPACK_DIR=repo_contents
+
+# Move files to root dir to avoid a License file error 
+mv $UNPACK_DIR/{AUTHORS,COPYING,COPYING.LESSER,LICENSE,README.md} .
+
+# Make the site-packages directory
+mkdir -p $SP_DIR/$PKG_NAME
+
+# Copy all of the appropriate things there
+cp -R $UNPACK_DIR/* $SP_DIR/$PKG_NAME
+cp {AUTHORS,COPYING,COPYING.LESSER,LICENSE,README.md} $SP_DIR/$PKG_NAME
+
+# Do the build in place in site-packages
+cd $SP_DIR/$PKG_NAME
+gpi_make --all --ignore-system-libs --ignore-gpirc -r 3
+
+# drop a version file with parseable info
+VERSION_FPATH=$SP_DIR/$PKG_NAME/VERSION
+echo "PKG_NAME: $PKG_NAME" > $VERSION_FPATH
+echo "PKG_VERSION: $PKG_VERSION" >> $VERSION_FPATH
+echo "GIT_FULL_HASH: $GIT_FULL_HASH" >> $VERSION_FPATH
```

Comment:
Who is setting this?

Reply:
I'm not sure what you're asking. I guess I should remove the last line since we're not sourcing from a Git repository anymore. The `PKG_NAME` and `PKG_VERSION` should be provided as conda environment variables, right?